### ISSUE TITLE
Minor error in lab-10-X1, backprop process

### ISSUE THREAD

**Issue by chaht01** [hunkim/DeepLearningZeroToAll#112]
I always appreciate with this repository and lecture video from youtube.
Unfortunately I think there is an error in back propagation step described in file lab-10-X1.

Line 43, `d_l2 = diff * sigma_prime(l2)`, I think it should be `d_l2 = diff`.

**Comment by kkweon** [comment]
Check #25

**Comment by chaht01** [comment]
Thanks to @kkweon !
I think it needs to notice what type of loss function is used(Same to #25 ).
(I feel sorry for my English)

**Comment by kkweon** [comment]
@chaht01 Yes, it's definitely in the milestones.
Can you create a new issue about, say, defining loss functions explicitly?
